client/src/modules/home/index.tsx

```diff
@@ -22,7 +22,11 @@ type User = NonNullable<AuthContextType['user']>;
 
 const Welcome = ({ user }: { user: User }) => {
   return (
-    <Flex alignItems={'center'} justifyContent="space-between">
+    <Flex
+      alignItems={'center'}
+      justifyContent="space-between"
+      paddingTop="20px"
+    >
```

Comment:
This is way better looking, thank you.

There is an issue with using padding, as it add spacing that latters can use, so languages like Japanese may have issues with weird flow with extra paddings

```suggestion
      marginTop="20px"
```